python/PyQt6/core/auto_generated/geometry/qgsabstractgeometry.sip.in

```diff
@@ -99,6 +99,28 @@ Constructor for QgsAbstractGeometry.
     virtual bool operator==( const QgsAbstractGeometry &other ) const = 0;
     virtual bool operator!=( const QgsAbstractGeometry &other ) const = 0;
 
+    virtual bool fuzzyEqual( const QgsAbstractGeometry &other, double epsilon = 1e-8 ) const = 0;
+%Docstring
+Performs fuzzy comparison between this geometry and ``other`` using an ``epsilon``.
+
+.. seealso:: :py:func:`fuzzyDistanceEqual`
+
+.. seealso:: :py:func:`QgsGeometryUtilsBase.fuzzyEqual`
+
+.. versionadded:: 3.36
+%End
+
+    virtual bool fuzzyDistanceEqual( const QgsAbstractGeometry &other, double epsilon = 1e-8 ) const = 0;
+%Docstring
+Performs fuzzy distance comparison between this geometry and ``other`` using an ``epsilon``.
```

Comment:
What is a "distance comparison"?

Reply:
This comes from https://github.com/qgis/QGIS/pull/55625 (and from the geometry checker plugin).

Traditionally, the comparison is done by examining the specific values (such as x and y) that define the location of points.  It focuses on the numerical differences or relationships between these values.

On the other hand, from the geometry checker plugin, comparing distances between points considers the actual spatial separation or length between the points, regardless of their coordinate values. This comparison involves measuring the distance between two points using formulas like the distance formula. Here, it's the "distance comparison".

Reply:
Would you mind adding those details to the docstrings?

Reply:
Done.